test/uk/gov/hmrc/transitmovementsrouter/controllers/MessageControllerSpec.scala

```diff
@@ -1086,4 +1088,58 @@ class MessageControllerSpec
         status(result) mustBe INTERNAL_SERVER_ERROR
     }
   }
+
+  "POST SDES callback" - {
+
+    lazy val movementId = MovementId("123e4567e89b12d3")
```

Comment:
Use these two with forAll and use arbitrary instead of checking with hardcoded value